corehq/apps/builds/models.py

```diff
@@ -32,7 +32,8 @@ class CommCareBuild(Document):
     build_number = IntegerProperty()
     version = SemanticVersionProperty()
     time = DateTimeProperty()
-    
+    j2me_enabled = BooleanProperty(default=False)
```

Comment:
From https://github.com/dimagi/commcare-hq/issues/12347 (emphasis mine):
> *Have HQ detect J2ME eligibility* by whether or not the build has the right attachments and save that to j2me_enabled (which _defaults to true_ for backwards compatibility).
It's important that it default to `True` because currently (before introducing these changes) _all_ builds were j2me enabled, so if the value is absent (i.e. if the build was made before this feature was introduced) then it should be `True`.